src/scripts/site.js

```diff
@@ -454,8 +454,6 @@ var AppInfo = {};
             define("apiclient", ["bower_components/emby-apiclient/apiclient"], returnFirstDependency);
         }
 
-        define("actionsheet", ["webActionSheet"], returnFirstDependency);
```

Comment:
why remove?

Reply:
Because webActionSheet is defined as actionSheet and is only used to define actionsheet. So, i fixed the define for actionsheet, removing webActionSheet as a actual module, because WebActionSheet and actionsheet is the same module. 

This looks confusing, but is stupid to begin with defining something like this.